bootstrap/crds/crds-sc.txt

```diff
@@ -1,10 +1,12 @@
 cert-manager/cert-manager.yaml
-prometheus-operator/alertmanager.yaml
-prometheus-operator/prometheus.yaml
-prometheus-operator/prometheusrule.yaml
-prometheus-operator/servicemonitor.yaml
+prometheus-operator/alertmanagerconfigs.yaml
+prometheus-operator/alertmanagers.yaml
+prometheus-operator/podmonitors.yaml
+prometheus-operator/probes.yaml
+prometheus-operator/prometheuses.yaml
+prometheus-operator/prometheusrules.yaml
+prometheus-operator/servicemonitors.yaml
 prometheus-operator/thanosrulers.yaml
```

Comment:
Should we move these into a folder called kube-prometheus-stack instead?
So it's less confusing to where the crds belong. (And the same for the wc-crds)

Reply:
I considered this as well, but I hesitated because the CRDs specifically are for prometheus-operator (as part of kube-prometheus-stack), and they're fetched from the [prometheus-operator repo](https://github.com/prometheus-operator/prometheus-operator/tree/master/example/prometheus-operator-crd). That said, maybe it's worth changing anyway for the sake of clarity?

Reply:
Great point! yea, then I think we should leave it as it is